services/ui-src/src/components/overlays/EntityDetailsOverlay.tsx

```diff
@@ -1,176 +1,123 @@
-import { useContext, useEffect, useState } from "react";
-import arrowLeftBlue from "assets/icons/icon_arrow_left_blue.png";
+import React, { MouseEventHandler, useContext, useEffect } from "react";
 // components
-import { Form, ReportContext, ReportPageIntro } from "components";
-import { Box, Button, Flex, Image, Text, Spinner } from "@chakra-ui/react";
+import { Box, Button, Flex, Image, Spinner, Text } from "@chakra-ui/react";
+import { Form, ReportPageIntro } from "components";
+// types
+import { EntityShape, EntityType, FormJson } from "types";
 // utils
-import {
-  AnyObject,
-  EntityShape,
-  EntityType,
-  FormJson,
-  isFieldElement,
-  ReportStatus,
-} from "types";
-import { filterFormData, useUser } from "utils";
+
+// assets
+import arrowLeftBlue from "assets/icons/icon_arrow_left_blue.png";
+// verbiage
 import accordionVerbiage from "../../verbiage/pages/accordion";
 import overlayVerbiage from "../../verbiage/pages/overlays";
 import { EntityContext } from "components/reports/EntityProvider";
+
 export const EntityDetailsOverlay = ({
+  closeEntityDetailsOverlay,
   entityType,
+  entities,
   form,
-  verbiage,
+  onSubmit,
   selectedEntity,
-  closeEntityDetailsOverlay,
-  setSidebarHidden,
+  disabled,
+  submitting,
   validateOnRender,
 }: Props) => {
-  const [submitting, setSubmitting] = useState<boolean>(false);
-  const { report, updateReport } = useContext(ReportContext);
-  const { full_name, state } = useUser().user ?? {};
-  const onError = () => {};
```

Comment:
Should be noted that this line in particular was causing huge errors we had no insight into. It was being passed into the form and hiding validation errors. I've removed the setting of a blank error handler and the validation errors we've been seeing should clear up.